src/components/Reports/FourteenMonthReports/Layout/Table/Table.test.tsx

```diff
@@ -169,57 +102,86 @@ const router = {
 };
 const onRequestSort = jest.fn();
 
-interface ComponentsProps {
+interface TestComponentProps {
   orderBy?: OrderBy | null;
 }
-const Components: React.FC<ComponentsProps> = ({ orderBy = 'name' }) => {
-  const mocks = transformedData();
-  const getTotals = () => mocks?.currencyGroups[0].totals.months || [];
 
+const InnerComponent: React.FC<Pick<TestComponentProps, 'orderBy'>> = ({
+  orderBy,
+}) => {
+  const { fourteenMonthReport } = useFourteenMonthReport(
+    'account-list-1',
+    FourteenMonthReportCurrencyType.Salary,
+  );
+
+  const currencyGroup = fourteenMonthReport?.currencyGroups[0];
+
+  if (!currencyGroup) {
+    return null;
+  }
+
+  return (
+    <FourteenMonthReportTable
+      isExpanded={true}
+      order="asc"
+      orderBy={orderBy ?? null}
+      orderedContacts={currencyGroup.contacts}
+      salaryCurrency={currencyGroup.currency}
+      onRequestSort={onRequestSort}
+      totals={currencyGroup.totals.months}
+    />
+  );
+};
+
+const Providers: React.FC<{ children: React.ReactElement }> = ({
```

Comment:
Since it's only used once, you can combine this `Providers` component with `TestComponent`. Just have `TestComponent` render all of the providers and `InnerComponent`.